### ISSUE TITLE
Notebook cell focus feedback should disappear when clicking the empty view content or somewhere else

### ISSUE THREAD

**Issue by joaomoreno** [microsoft/vscode#121440]
I finally figured out the details of something that annoys me all the time our notebooks. Notice in the GIF how that border of the focused cell doesn't disappear when clicking elsewhere... I can only make it disappear when pressing <kbd>Escape</kbd>.

![recording (6)](https://user-images.githubusercontent.com/22350/114997392-a35aa400-9e54-11eb-8b13-e3d7f42ec9fe.gif)

**Comment by misolori** [comment]
I would agree, very similar to the explorer that selecting an empty area shifts focus to the entire view. Though not sure what we'd change the focus to in the editor?
